ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
 ETC_20150317_000097	$Yellow Griba
 ETC_20150317_000098	$Large Red Griba
 ETC_20150317_000099	$Big Griba
-ETC_20150317_000100	Someone once described this monster as hard to block and hard to eat.
+ETC_20150317_000100	$Someone once described this monster as hard to block and hard to eat.
```

Comment:
"This monster was once described as hard to block and hard to eat."

Big Griba made the subject.